README.md

```diff
@@ -27,6 +27,38 @@ npm install eslint eslint-plugin-package-json jsonc-eslint-parser --save-dev
 
 ## Usage
 
+### Flat Config
+
```

Comment:
[Docs] Since the new config does a lot more for users, I think we'll want to explain a bit of it to them. Maybe:

```suggestion

This plugin's recommended configuration enables its rules on `**/package.json` files, parsing them with [`jsonc-eslint-parser`](https://github.com/ota-meshi/jsonc-eslint-parser):

```